recipes/libblitz/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+./configure \
+  --prefix="${PREFIX}" \
+  --enable-static \
+  --enable-shared \
+  --enable-optimize \
+  --enable-64bit \
```

Comment:
So, there are some 32-bit platforms. We should be checking [`ARCH`]( http://conda.pydata.org/docs/building/environment-vars.html#environment-variables-set-during-the-build-process ) and setting this flag accordingly.